### PR TITLE
added TDCNpp

### PR DESCRIPTION
**Pull request by popcornell**
we might want to add TDCN++ from Universal Sound Separation and FUSS

### CODE REVIEW

File: asteroid/masknn/convolutional.py [hunk 1]
```diff
@@ -211,6 +211,448 @@ def get_config(self):
...
+            "n_blocks": self.n_blocks,
+            "n_repeats": self.n_repeats,
+            "n_src": self.n_src,
+            "norm_type": self.norm_type,
+            "mask_act": self.mask_act,
+        }
+        return config
+
+
+class SuDORMRF(nn.Module):
```
Comment (mpariente) [inline asteroid/masknn/convolutional.py:386]: Can we have the code of TDCNpp only? There are few other things in this commit.

**Review by mpariente — COMMENTED** [review]
Apart from the code which doesn't come from the PR, it looks good. 
We would need a unit test as well please.

File: asteroid/masknn/convolutional.py (hunk 1 above)
Comment (popcornell) [reply asteroid/masknn/convolutional.py:386]: SuDORMRF was not merged ?

**Comment by mpariente** [comment]
`sudo rm -rf SuDORMRF network` :rofl:

**Comment by mpariente** [comment]
There is still a problem with the PR, sorry. We should have rebased master.

**Comment by mpariente** [comment]
The SuDORMRF code was duplicated, I removed it. Fix two small things and added tests. 
Ready to merge on my side.

**Pull request merged**